From 3d7b5b0faf63c5634ebee922aa3d24cb718d6aac Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Elan=20Ruusam=C3=A4e?= <glen@delfi.ee>
Date: Sun, 3 Oct 2010 14:00:23 +0300
Subject: [PATCH] - fix missing include for local graphviz install. ticket #1

---
 syntax.php |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/syntax.php b/syntax.php
index 2102b5a..6cffd11 100644
--- a/syntax.php
+++ b/syntax.php
@@ -9,6 +9,7 @@
 
 if(!defined('DOKU_INC')) define('DOKU_INC',realpath(dirname(__FILE__).'/../../').'/');
 require_once(DOKU_INC.'inc/init.php');
+require_once(DOKU_INC.'inc/media.php');
 if(!defined('DOKU_PLUGIN')) define('DOKU_PLUGIN',DOKU_INC.'lib/plugins/');
 require_once(DOKU_PLUGIN.'syntax.php');
 
-- 
1.7.2.2

